libs/dh/wholesale/feature-calculations/src/lib/details/details.component.ts

```diff
@@ -68,19 +83,49 @@ import { VaterFlexComponent, VaterUtilityDirective } from '@energinet-datahub/wa
   styleUrls: ['./details.component.scss'],
 })
 export class DhCalculationsDetailsComponent {
+  private toast = inject(WattToastService);
+  private transloco = inject(TranslocoService);
   private apollo = inject(Apollo);
-  private injector = inject(Injector);
 
   @Output() closed = new EventEmitter<void>();
   id = input<string>();
 
   @ViewChild(WattDrawerComponent)
```

Comment:
Suggetion: if it does make the the PR much bigger, maybe consider changing to signal format as well

Reply:
Good suggestion, I already have some work on that in a different branch, but it is not completely trivial since I also have to refactor the graphql request. So I think it is best to do it in a different PR.

Reply:
c31dce7a64b53741addbf039f29075b65313f647